SignalServiceKit/src/Messages/OWSOutgoingNullMessage.m

```diff
@@ -35,7 +36,7 @@ - (instancetype)initWithContactThread:(TSContactThread *)contactThread
 
 #pragma mark - override TSOutgoingMessage
 
-- (NSData *)buildPlainTextData
+- (NSData *)buildPlainTextData:(SignalRecipient *)recipient
 {
     OWSSignalServiceProtosContentBuilder *contentBuilder = [OWSSignalServiceProtosContentBuilder new];
```

Comment:
Rather than remembering to add the `[self addLocalProfileKeyIfNecessary:recipient]` to every new content type, maybe we should consider wrapping the Content builder.